### ISSUE TITLE
other graphs show jitsi video bridge connecting with prosody and not with cofo

### ISSUE THREAD

**Issue by f1-outsourcing** [jitsi/docker-jitsi-meet#266]
Other graphs show jitsi video bridge connecting with prosody and not with cofo

https://github.com/jitsi/docker-jitsi-meet

**Comment by saghul** [comment]
That's correct, the JVB connects to Prosody. Jicofo and the JVB communicate via Prosody. I'll update the graph.
